flaskerk/converters.py

```diff
@@ -0,0 +1,102 @@
+def get_converter_type_any(*args, **kwargs):
+    """
+    Handle converter type "any"
+    :param args:
+    :param kwargs:
+    :return: return schema dict
```

Comment:
It seems the `sphinx` cannot generate the doc properly if there is no newline between description and parameters.

By the way, can you help me add this part to the `docs` so it can be generated by `sphinx` during the CI.

Reply:
> It seems the `sphinx` cannot generate the doc properly if there is no newline between description and parameters.
> 
> By the way, can you help me add this part to the `docs` so it can be generated by `sphinx` during the CI.

Added newlines :)